Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -53,7 +53,6 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
-		FF098EFA2AB3424E003EC0FE /* Basic-Car-Maintenance.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = "Basic-Car-Maintenance.xcconfig"; sourceTree = SOURCE_ROOT; };
```

Comment:
This shouldn't be removed, please follow the instructions again in the [README](https://github.com/mikaelacaron/Basic-Car-Maintenance/tree/main#getting-started) for properly setting up the project.

Reply:
@ayushs-2k4 This file is still being changed, and it should not be changed at all.

See [this video](https://www.youtube.com/live/60ivxAOUrYU?si=NX8hVyb55QQUo-Gs&t=1667) at `27:47`